packages/mds-api-authorizer/tests/index.spec.ts

```diff
@@ -9,13 +9,13 @@ const ADMIN_AUTH = `basic ${Buffer.from(`${PROVIDER_UUID}|${PROVIDER_SCOPES}`).t
 describe('Test API Authorizer', () => {
   it('No Authorizaton', done => {
     const authorizer = AuthorizationHeaderApiAuthorizer({} as express.Request)
-    test.value(authorizer).is({})
+    test.value(authorizer).is(null)
```

Comment:
Just curious, what's the motivation for this change?

Reply:
Makes the type guarding more predictable because it forces authorizers to return claims and not partial claims and is more reflective of what actually happens in the authorizer.

Reply:
👍